util/spell.sh

```diff
@@ -0,0 +1,48 @@
+#!/bin/bash
+
+if ! which -s aspell; then
+  echo 'aspell command not found, cannot run spell check'
+  exit
+fi
+
+ROOT=`git rev-parse --show-toplevel`
+
+# This gets us only english .md files at the moment, but is clearly brittle
+MARKDOWN_FILES=`find $ROOT -name '*.md' -maxdepth 2 -not -name '*-ptBR.md'`
```

Comment:
I believe we also have Japanese docs somewhere in the tree; this finder line needs to at least filter those out.